### DISCUSSION TITLE
How to strip `www.` from filenames?

### DISCUSSION THREAD

**Discussion by user-1-2-3-4-5** [gildas-lormeau/SingleFile#1319] [Q&A] [answered]
Can the filename template be edited to remove the [WWW prefix](https://en.wikipedia.org/wiki/World_Wide_Web#WWW_prefix)?

Note that I find subdomains as valuable information, I only want to disable the leading `www.` string from being saved (e.g. `www.example.com` → `example.com`), when used by a domain.

**Comment by gildas-lormeau** [comment, marked as answer]
The template `{url-hostname-subdomains}%if-not-empty<{url-hostname-subdomains}|.>{url-hostname-root}` should return the value you wish.

**Comment by user-1-2-3-4-5** [comment]
my full currently used option, for the record ('file name' → template`):

    webpages/{url-hostname-subdomains}%if-not-empty<{url-hostname-subdomains}|.>{url-hostname-root}-%if-empty<{page-title}|No title>.({date-locale}).html

Saves to the sub-folder `webpages` in `~/Downloads`.
